pandas/io/parsers/readers.py

```diff
@@ -195,9 +195,9 @@
     Dict of functions for converting values in certain columns. Keys can either
     be integers or column labels.
 true_values : list, optional
-    Values to consider as True.
+    Values to consider as True in addition to case-insensitive variants of "True".
```

Comment:
Do you know if there are tests for case-insensitive variants of `True/False`?

Reply:
That's a great question, it will tell us whether the case insensitivity is intended or unnoticed behavior. I'll take a look today or tomorrow.

Reply:
@mroeschke I've taken a look, and the closest that I can find is `test_boolean_dtype` [here](https://github.com/pandas-dev/pandas/blob/c2fade19bbf275f1fe5022f1c9ded387e52a3921/pandas/tests/io/parser/dtypes/test_dtypes_basic.py#L130). There are also some conversion tests `test_na_values`. But those have a different purpose.

What would your suggestion be?